lms/db/__init__.py

```diff
@@ -81,8 +81,10 @@ def update_from_dict(self, data, skip_keys=None):
 SESSION = sessionmaker()
 
 
-def init(engine):
+def init(engine, drop=False):
     """Initialise the database tables."""
```

Comment:
Doc string for the param here? Seems like an obvious, but important one.

Reply:
Added one